src/app/code/community/SchumacherFM/OpCachePanel/Model/Types/OpCache.php

```diff
@@ -69,7 +87,8 @@ public function compile($pathToFile)
     {
 
         $files = $this->_getCompiledFiles();
-        if (isset($files[$pathToFile])) { // work around to avoid a segmentation fault in php binary :-(
+        $blackList = $this->_getBlackList();
+        if (isset($files[$pathToFile]) || in_array($pathToFile, $blackList)) { // work around to avoid a segmentation fault in php binary  and skip files in opCache black list
```

Comment:
remove in_array and use isset, see comment above regarding http://php.net/manual/en/function.array-flip.php